pkg/util/predicate/BUILD.bazel

```diff
@@ -8,7 +8,7 @@ go_library(
         "generic.go",
         "predicate.go",
     ],
-    importpath = "github.com/jetstack/cert-manager/pkg/controller/certificates/internal/predicate",
+    importpath = "github.com/jetstack/cert-manager/pkg/util/predicate",
     visibility = ["//pkg/controller/certificates:__subpackages__"],
```

Comment:
This `visibility` declaration doesn't get auto-updated by our code generation stuff 😞  so this should be updated to `visibility = ["//visibility:public"],` manually

Reply:
(otherwise other packages won't actually be able to import this package when built with bazel.....)

Reply:
I knew there was something I forgot! Had this problem before..